packages/mwp-config/locales.js

```diff
@@ -0,0 +1,16 @@
+module.exports = [
```

Comment:
@willh-meetup this and the `localesSecondary` file below are where the locale config will be coming from going forward, although I'd like to organize things a little better at some point - I'm wondering if we might want have a module that exports `{ primary: [], secondary: [], beta: [] }` or something.

Reply:
Yeah, that's a good call. Would be nice to export as a single module, since they are often used in relation to eachother.